components/input.css

```diff
@@ -0,0 +1,73 @@
+.input-box {
+	width: 100%;
+	display: flex;
+	position: relative;
+	font-size: 1.5rem;
+}
+.input__box-icon {
+	padding: 1.5rem 0.7rem;
+	align-self: center;
+	position: absolute;
+	left: 0.2rem;
```

Comment:
Instead of 0.2rem it would be better if you could use px. Anything that sums up to <=10 px should be preferably in px rather than responsive units.

Reply:
Okay, I'll update it!